packages/elements/src/internals/machines/form/form.machine.ts

```diff
@@ -107,10 +110,13 @@ export const FormMachine = setup({
 
           for (const field in fields) {
             enqueue({
-              type: 'setFieldErrors',
+              type: 'setFieldFeedback',
               params: {
                 name: field,
-                errors: fields[field],
+                feedback: {
+                  type: 'error',
+                  message: fields[field][0],
```

Comment:
Since in `FieldError` we only accessed the first error anyways, this shortcuts it and sets it here already